src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -96,7 +96,7 @@ String delete(String uid) throws Exception {
     }
 
     /**
-     * Deletes an index if exists in the MeiliSearch instance
+     * Deletes an index if exists in the Meilisearch instance
```

Comment:
This function should have been removed I think but that's not the point of this PR 😅

Reply:
This package is still not update unfortunately.
The PR who delete it is [here](https://github.com/meilisearch/meilisearch-java/pull/307)

Reply:
My bad !